corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -463,13 +463,20 @@ def add_frame(self, frame):
         self.node.append(frame.node)
 
 
+class StackJump(XmlObject):
+    ROOT_NAME = 'jump'
+
+    url = NodeField('url/text', Text)
+
+
 class Argument(IdNode):
     ROOT_NAME = 'argument'
 
 
 class SessionEndpoint(IdNode):
     ROOT_NAME = 'endpoint'
 
+    command_id = StringField('@command_id')
```

Comment:
it this still used? I don't see it in any of the test XML and thought I recalled you removing it.

Reply:
oops, bfc6710f02615389d2e27a7e1bfc8438d66c778e